client/python/README.md

```diff
@@ -1,19 +1,73 @@
+4) Install a single Python version using Pyenv by running `pyenv install [python version]`.
+5) Install multiple Python versions using Pyenv by running `pyenv install --list`.
+6) Install [poetry](https://python-poetry.org/) on your local machine by following the instructions on [python poetry documentation](https://python-poetry.org/docs/).
+
+On Mac Machine
+=
+
+1) Open your favorite shell/package manager, for example, [Homebrew](https://brew.sh/) or [MacPorts](https://www.macports.org/).
+2) Install [Git](https://git-scm.com/) by running `$ brew install git` or `$ sudo port install git` or follow the instructions [here](https://www.atlassian.com/git/tutorials/install-git)
+7) Fork the G-Research Armada repository to your personal repository.
+8) Clone the forked armada repository on your personal git repository by running git clone [Armada Repo](https://github.com/G-Research/armada.git) and navigate into the new directory Armada project by running cd armada.
 
+### Python / Poetry and Pyenv installation
+#### Note that this repository does not support python 3.10 for now
 
+1) Install [Pyenv](https://github.com/pyenv/pyenv) by running `$ brew update` `$ brew install pyenv` then follow the steps on [Set up your shell environment for Pyenv](https://github.com/pyenv/pyenv#set-up-your-shell-environment-for-pyenv) for post-installation instructions.
+2) Install a single Python version using Pyenv by running `pyenv install [python version]`.
+3) Install multiple Python versions using Pyenv by running `pyenv install --list`.
+4) Install [poetry](https://python-poetry.org/) on your local machine by following the instructions on [python poetry documentation](https://python-poetry.org/docs/).
+
+On Windows Machine
+=
+### Note: This repository can only run on Windows Subsystem for Linux (WSL)
+
+1) To install Windows Subsystem for Linux on your local machine follow the instructions [here](https://docs.microsoft.com/en-us/windows/wsl/install) 
+2) Install [Git](https://git-scm.com/) by running `$ sudo apt-get install git` or follow the instructions [here](https://docs.microsoft.com/en-us/windows/wsl/tutorials/wsl-git)
+3) Fork the G-Research Armada repository to your personal repository.
+4) Clone the forked armada repository in your personal git repository by running git clone [Armada Repo](https://github.com/G-Research/armada.git) and navigate into the new directory Armada project by running cd armada.
+
+### Python / Poetry and Pyenv Installation
+#### Note that this repository does not support python 3.10 for now
+
+1) Install [Pyenv](https://github.com/pyenv/pyenv) by running `sudo apt-get update` to update and `sudo apt-get install make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev` to get all pyenv dependencies.
```

Comment:
As with above, a single multi-line code block here would be easier to read/cut/paste.